packages/common/src/eips/4844.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "EIP-4844",
+  "number": 4844,
+  "comment": "Shard Blob Transactions",
+  "url": "https://eips.ethereum.org/EIPS/eip-4844",
+  "status": "Draft",
+  "minimumHardfork": "shanghai",
```

Comment:
I should really document this somewhere since nearly everyone is stumbling upon this 😋: this is *not* the HF where the EIP is likely to be included but the one which is necessary to run the EIP (in doubt: in isolation). So in this case this would be `Merge`.

Reply:
Technically it is not merge? It is London, that one introduced 1559, Berlin introduced 2718 and 2930.

Reply:
Yeah, one can also take lower HF choices, `Merge` is just the most lazy one.

The earlier one goes, the more the risk grows that one might overlook some technical interdependencies which causes things not to work. On the other hand one get a bit more flexibility on EIP-combination runs.

I have some tendency to mostly go for these lazy choices, I would assume use cases where people who want to run something pre-Merge and have EIP-4844 activated are pretty pretty limited respectively eventually non-existant (?).

And in doubt this is easy to adjust, so my philosphy suggestion here would be "start high and eventually go lower" if someone opens an issue "but I want to run EIP-4844 on Berlin!". 🙂 

Anyhow: again one of this lower-level-important choices. There can also not much go wrong if we put in `berlin` or so here, just some personal decision guidance I find practically useful.

Reply:
I'm reasonably confident from reading the EIP spec that we shouldn't allow this EIP to require any less than `merge`.  The entire EIP is premised on the EL just passing the blobs to the CL with minimal verification done at the EL level.

Reply:
I have no strong opinion here, but if we detect technical inconsistencies if we choose the "non-lazy"fork then it means the EIP is incomplete, since it would miss EIPs which it would depend upon (which would thus lead to updating the EIP dependencies which would be healthy for the EIP ecosystem). But I don't want to start bike shedding here, `merge` is fine :)